pkgs/misc/emulators/dolphin-emu/master.nix

```diff
@@ -18,29 +18,29 @@ let
     categories = "Game;Emulator;";
     startupNotify = "false";
   };
-in stdenv.mkDerivation rec {
+in mkDerivation rec {
```

Comment:
By using `mkDerivation` in automatically includes `wrapQtAppsHook` which brings `qtbase` as a build time dependency. Can you use `stdenv.mkDerivation` and add `wrapQtAppsHook` like?
```
nativeBuildInputs = [
...
] ++ lib.optional stdenv.isLinux wrapQtAppsHook;
```